observation_portal/proposals/models.py

```diff
@@ -232,7 +232,7 @@ class Membership(models.Model):
     PI = 'PI'
     CI = 'CI'
     ROLE_CHOICES = (
-        (PI, 'Pricipal Investigator'),
```

Comment:
Can you create a new migration for this change?

Reply:
Done 👍